js/MineSweeper.js

```diff
@@ -478,6 +516,7 @@ jQuery(function ($) {
 
             $('#mine_flag_display').val(numMines);
             $('#timer').val(0);
+            $('#pause').hide();
```

Comment:
Just so I understand, when is it right to use an ID and when should we uses classes to identify elements we have created?  I personally would use a class when I have more than one element I wish to address but an ID when I have only the one, so in this case there is only one Pause button hence I chose the ID (much as the existing code uses '#level' for the level selector).  Just trying to understand, happy to change it :)